test/foundry/mocks/hts-precompile/HederaNonFungibleToken.sol

```diff
@@ -158,8 +152,8 @@ contract HederaNonFungibleToken is ERC721, Constants {
         return uint64(nftCount.minted - nftCount.burned);
     }
 
-    function isApprovedOrOwner(address spender, uint256 tokenId) external view returns (bool) {
-        return _isApprovedOrOwner(spender, tokenId);
+    function isApprovedOrOwner(address owner, address spender, uint256 tokenId) external view returns (bool) {
```

Comment:
Q: why this change was something broken or is the name wrong?
Even if the change is needed should it be an additional app or is it appropriate to modify?

Reply:
So the `_isAuthorized()` method on line 156 is from OZ/ERC721.sol. In the upgrade, OZ added a new argument for the `_isAuthorized()` method that's why it has this change. I'm not quite clear on what you're asking for here 
> Even if the change is needed should it be an additional app or is it appropriate to modify?

Reply:
Okay, great, mixed it up for something else. LG